From 6c1a87b12f565c706a58387b1d7a28cf4efa4cb1 Mon Sep 17 00:00:00 2001
From: Billy Tsai <billy_tsai@aspeedtech.com>
Date: Wed, 30 Mar 2022 15:16:56 +0800
Subject: [PATCH] pci: Remove the reset assert from probe function.

Because PCIe RC/EP reset short together, so the driver can't assert the
reset after BMC reboot which will lead to x86 RC can't find the Ast2600
EP.

Signed-off-by: Billy Tsai <billy_tsai@aspeedtech.com>
Change-Id: Iab8ccb65c893df93409d66c3a97938062e62c408
---
 drivers/pci/aspeed_pcie_phy.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/pci/aspeed_pcie_phy.c b/drivers/pci/aspeed_pcie_phy.c
index ddd7dc4a25..c33c502a8e 100644
--- a/drivers/pci/aspeed_pcie_phy.c
+++ b/drivers/pci/aspeed_pcie_phy.c
@@ -74,8 +74,6 @@ static int aspeed_pcie_phy_probe(struct udevice *dev)
 		return ret;
 	}
 
-	//reset rc bridge
-	reset_assert(&reset_ctl);
 	reset_deassert(&reset_ctl);
 
 	rc_bridge->reg = devfdt_get_addr_ptr(dev);
-- 
2.25.1

